madgraph/iolibs/export_v4.py

```diff
@@ -7027,10 +7033,17 @@ def create_coupl_inc(self):
 
         # Write header
         header = """C
-C NB: VECSIZE_MEMMAX is defined in vector.inc
-C NB: vector.inc must be included before coupl.inc
+C     NB: VECSIZE_MEMMAX is defined in vector.inc, but this is not
+C     included in coupl.inc because Fortran has no include guards
+C
+C     NB: therefore coupl.inc introduces the definition of a separate
+C     variable VECSIZE_MEMMAX_COUPL, whose value must be the same as
+C     that of VECSIZE_MEMMAX (this is cross-checked in PROGRAM DRIVER)
 C
```

Comment:
Ok, I see now, i would prefer the previous option is to use 
include 'vector.inc' before each line include 'coupl.inc'
with a comment on why the include to vector.inc is needed